.github/workflows/ci.yml

```diff
@@ -22,15 +22,8 @@ jobs:
 
       - name: Run script to validate commits for both pull request and a push
         env:
-          PY_COLORS: '1'
-          ANSIBLE_FORCE_COLOR: '1'
-          GITHUB_PULL_REQUEST: ${{ github.event.number }}
-          GITHUB_PULL_REQUEST_BODY: ${{ github.event.pull_request.body }}
-          GITHUB_BRANCH: ${{ github.head_ref }}
-          GITHUB_REPO_SLUG: ${{ github.repository }}
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          GITHUB_CONTEXT: ${{ github.event.pull_request.commits_url }}
           GITHUB_PR_COMMITS_URL: ${{ github.event.pull_request.commits_url }}
+          GITHUB_USER: ${{ github.event.pull_request.user.login }}
```

Comment:
we can add more vars in 2 ways

1. https://github.com/ansible/galaxy_ng/blob/master/template_config.yml#L26 adding as a config param
2. add as part of template https://github.com/pulp/plugin_template/blob/master/templates/macros.j2#L1-L18